src/applications/vaos/new-appointment/components/FormLayout.jsx

```diff
@@ -28,17 +29,27 @@ function Title() {
   return 'New appointment';
 }
 
-export default function FormLayout({ children, isReviewPage, pageTitle }) {
+export default function FormLayout({ children, isReviewPage, pageKey }) {
   const location = useLocation();
+  const history = useHistory();
+  const dispatch = useDispatch();
   return (
     <>
       <MhvSecondaryNav />
       <div className="vads-l-grid-container vads-u-padding-x--2p5 large-screen:vads-u-padding-x--0 vads-u-padding-bottom--2">
-        <Breadcrumbs>
-          <a href="/my-health/appointments/schedule/type-of-care">
-            {pageTitle}
-          </a>
-        </Breadcrumbs>
+        {location.pathname.endsWith('type-of-care') ? (
+          <Breadcrumbs />
+        ) : (
+          <va-link
+            onClick={() =>
+              dispatch(routeToPreviousAppointmentPage(history, pageKey))
+            }
+            text="Back"
+            data-testid="schedule-back-link"
+            tabindex="0"
+          />
+        )}
```

Comment:
The NavLink didn't work for me so I used the `va-link` web component instead. From there, replicated the back form button's use of `routeToPreviousAppointmentPage`. That way the back link and back button are sharing the same history.